scripts/tests/biastests_step1.py

```diff
@@ -0,0 +1,111 @@
+import os
+from utilities import logging
+import argparse 
+
+logger = logging.setup_logger(__file__, 3, True)
+
+# Perform bias tests on different (pseudo) data sets. Use one dataset + uncertainties as nominal and fit the other set as pseudodata.
+#   The shift in the mass parameter is the bias
+# This script is step 1 and done within the WRemnants singularity and produces the histograms and combine inputs
```

Comment:
As discussed would be a good idea to have a shell driver script that runs all from the right singularity. Can use singularity run as here: https://github.com/WMass/WRemnants/blob/main/.github/workflows/main.yml#L31

Reply:
In principle a good idea. But I think the biastest scripts are a bit fragile, I don't think it will be beneficial to fully automatize at this stage